chroma-dependencies/Makefile

```diff
@@ -63,7 +63,7 @@ requirements::
 	echo "tzlocal";                                           \
 	echo "systemd-python";                                    \
 	echo "iml-common>=1.0,<1.1";                              \
-	echo "https://dl.dropboxusercontent.com/s/qqjud11bvh8vo68/coverage-3.5.2pl2.tar.gz"; \
+	echo "coverage==3.5.2"; \
```

Comment:
Only because there is effectively no time invested in testing yet, should line the continuator with the previous line.